.gitignore

```diff
@@ -48,3 +48,7 @@ Thumbs.db
 # DB data
 /db/data/*
 !/db/data/*.sample
+
+# Javascript files
```

Comment:
The generated files from typescript source files. For example webstorm with some configurations could expose them. I think would be better as a preventive measure.